flink-dist/pom.xml

```diff
@@ -573,7 +573,7 @@ under the License.
 									<pattern>py4j</pattern>
 									<shadedPattern>org.apache.flink.api.python.py4j</shadedPattern>
 									<includes>
-										<include>py4j.*</include>
+										<include>py4j.*.*</include>
```

Comment:
Why do we need to add this change?

Reply:
yes, without this change py4j is not shaded completely.

Reply:
May be `net.sf.py4j:*` is correct, and `FLINK-12409` will correct this change.

Reply:
Yes, the solution in `FLINK-12409` makes more sense. I have removed this change in the new commit which will cause the CI test failure for the moment. I will rebase this PR after `FLINK-12409` merged and this problem would be solved.